plugins/aws/iam/rootMfaEnabled.js

```diff
@@ -52,7 +52,9 @@ module.exports = {
             }
         }
 
-        if (!found) {
+        if (settings.govcloud && !found) {
```

Comment:
Do we need to check for !found? instead we can check only for govcloud and that is enough is not it?

Reply:
Yes that is enough. I just wanted to be sure that it does not produce two results. What do you think?